x-pack/plugins/security_solution/public/detections/pages/detection_engine/rules/create/helpers.ts

```diff
@@ -537,15 +541,18 @@ export const formatAboutStepData = (
 export const formatActionsStepData = (actionsStepData: ActionsStepRule): ActionsStepRuleJson => {
   const {
     actions = [],
+    responseActions,
     enabled,
     kibanaSiemAppUrl,
     throttle = NOTIFICATION_THROTTLE_NO_ACTIONS,
   } = actionsStepData;
 
   return {
     actions: actions.map(transformAlertToRuleAction),
+    response_actions: responseActions?.map(transformAlertToRuleResponseAction),
     enabled,
-    throttle: actions.length ? throttle : NOTIFICATION_THROTTLE_NO_ACTIONS,
+    throttle:
+      actions.length || responseActions?.length ? throttle : NOTIFICATION_THROTTLE_NO_ACTIONS,
```

Comment:
If I understood correctly, throttle only affects `actions`. So why `responseActions` have any impact on which throttle can be shown in the UI?

Reply:
Remove the connection between throttle and respponseActions 👍